docs/models.yml

```diff
@@ -212,18 +212,23 @@ user:
     fields:
       type: relation-list
       to: projector/current_element_ids
-  vote_delegated_$_to_id:
```

Comment:
Please undo this change! The template form is required for requesting specific data from the autoupdate service. The format cannot deal with m2m tables spanning over meetings.

Reply:
@ostcar Please discuss this during code sprint. Only to be kept in mind: The problem has something to do with the structure of permission service (only yes or no) and restriction service (restrict relation id fields only if the user can not see any field of the related object). @FinnStutzenstein may give an example of an autoupdate request that requires this template form tomorrow.

Reply:
I removed the commit